modules/RTC/DataChannels.js

```diff
@@ -247,4 +239,37 @@ DataChannels.prototype._some = function (callback, thisArg) {
     }
 };
 
+/**
+ * Sends passed object via the first found open datachannel
+ * @param jsonObject {object} the object that will be sent
+ */
+DataChannels.prototype.send = function (jsonObject) {
```

Comment:
Can we log a message if sending fails because the channel is not open? This could mask problems and it is a known issue that data channels sometimes open with a 3 second delay (because of a bug in the bridge).